index.html

```diff
@@ -2,77 +2,114 @@
-                        <li data-submenu="#submenu1_1">
-                            <a href="#">Menu 1 sub 1</a>
+        <div class="all_demos">
+            <h1>
+                Uniter PHP<br/>in Browser Demos
+            </h1>
+            <p>
+                The in-browser logic for all these pages is written entirely in PHP,
+                see <a target="_blank" href="//github.com/PharaohTools/uniter-jquery/blob/master/php/src/">Demos</a>
+                for a look at the php code.<br /> All Javascript and JQuery object and method calls come directly from php code.
+            </p>
+            <p>
+                The tests are written using PHPUnit and run on the command line (for now),
+                see <a target="_blank" href="//github.com/PharaohTools/uniter-jquery/blob/master/tests/phpunit/Demo/Tests/Component/NavMenuComponentTest.php">NavMenuComponentTest</a>.
+            </p>
+            <p>
+                (Almost) No JavaScript was used in the making of these demos :)
+            </p>
+        </div>
 
-                            <ol class="submenu" id="submenu1_1">
-                                <li data-submenu="#submenu1_1_1">
-                                    <a href="#">Menu 1 sub 1 sub 1</a>
+        <div class="all_demos" id="homepage_loading">
+            <img src="img/loading.gif" />
+        </div>
 
-                                    <ol class="submenu" id="submenu1_1_1">
-                                        <li>
-                                            <a href="#">Menu 1 sub 1 sub 1 sub 1</a>
-                                        </li>
-                                        <li>
-                                            <a href="#">Menu 1 sub 1 sub 1 sub 2</a>
-                                        </li>
-                                    </ol>
-                                </li>
-                                <li>
-                                    <a href="#">Menu 1 sub 1 sub 2</a>
-                                </li>
-                            </ol>
-                        </li>
-                        <li>
-                            <a href="#">Menu 1 sub 2</a>
-                        </li>
-                    </ol>
-                </li>
-                <li data-submenu="#submenu2">
-                    <a href="#">Menu 2</a>
-                    <ol class="submenu" id="submenu2">
-                        <li>
-                            <a href="#">Menu 2 sub 1</a>
-                        </li>
-                        <li>
-                            <a href="#">Menu 2 sub 2</a>
-                        </li>
-                    </ol>
-                </li>
-            </ol>
-        </nav>
+        <div class="all_demos" id="homepage_slider">
+            <div class="demo_entry">
+                <p>Hello World!</p>
+                <a target="_blank" href="/hello_world.html">Hello World</a>
```

Comment:
As above, these paths need to be relative so that they can work on eg. GitHub pages